### PR TITLE
RHDEVDOCS 5669 use a variable for Pipelines version in links, enable ARM64 tkn for all docs

### PR DESCRIPTION
**Pull request by mramendi**
Version(s):

please cp to pipelines-docs-1.12

Issue:

RHDEVDOCS 5669

Link to docs preview:

https://66520--docspreview.netlify.app/openshift-pipelines/latest/tkn_cli/installing-tkn
https://66520--docspreview.netlify.app/openshift-pipelines/latest/about/about-pipelines
https://66520--docspreview.netlify.app/openshift-pipelines/latest/pac/install-config-pipelines-as-code#installing-pipelines-as-code-cli_install-config-pipelines-as-code
https://66520--docspreview.netlify.app/openshift-pipelines/latest/install_config/installing-pipelines#op-installing-pipelines-operator-in-web-console_installing-pipelines

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/op-installing-tkn-on-linux-using-rpm.adoc [hunk 1]
```diff
@@ -48,33 +48,31 @@ For {op-system-base-full} version 8, you can install the {pipelines-title} CLI a
...
 ----
-ifndef::openshift-rosa,openshift-dedicated[]
 +
 * Linux on ARM (aarch64, arm64)
 +
-[source,terminal]
+[source,terminal,subs="attributes"]
 ----
-# subscription-manager repos --enable="pipelines-1.11-for-rhel-8-arm64-rpms"
+# subscription-manager repos --enable="pipelines-{pipelines-version-number}-for-rhel-8-arm64-rpms"
```
Comment (VeereshAradhya) [inline modules/op-installing-tkn-on-linux-using-rpm.adoc:74]: It should be `pipelines-1.12-for-rhel-8-aarch64-rpms`.

**Review by concaf — APPROVED** [review]

**Review by VeereshAradhya — APPROVED** [review]

**Comment by mburke5678** [comment]
@mramendi I am happy to review these changes; however, it would help to have direct preview links to each changed file, or some indication of where they are in the book, so that I can test the links.

**Comment by mramendi** [comment]
@mburke5678 I have added links to those of the files that are not under the first link I provided.

**Comment by mburke5678** [comment]
@mramendi Thanks for the links. Maybe it is just me, but I was having a hard time finding the files! /lgtm

**Pull request merged**